Study_repos/HomeWorks/HW03.Birthday/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace HW03.Birthday
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текущий год");
+            int cr_year = Convert.ToInt32(Console.ReadLine());
```

Comment:
Suggestion - DO NOT use `_` in variables between two words. just use as below:

`crYear' instead of 'cr_year'